src/components/App/App.vue

```diff
@@ -0,0 +1,35 @@
+<template>
+  <div>
+    <!-- <p>{{debug(cities)}}</p> -->
+    <Header />
+    <Form :cities="cities" :states="states" />
+    <Container />
+  </div>
+</template>
+
+<script>
+import Header from '../Header/Header'
+import Form from '../Form/Form'
+import Container from '../Container/Container'
+import data from '../../data/location-data'
+
+export default {
+  name: 'App',
+  components: {
+    Header,
+    Form,
+    Container
+  },
+  data: () => ({
```

Comment:
Glad we figured this out in the end! Looking forward to moving on to bringing in the API data instead.